components/healthcare/code/healthcare-bbe/powerful-edi-for-claims.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Powerful EDI for claims'
-description: Ballerina has powerful support for Electronic Interchange Formats (EDI). In healthcare, this means out of the box handling of X12, 834s, 837s and more region specific standards via native schema mapping. Go from EDI formats to Ballerina Records and back; Do Data Mapping on that and you’re off to the races!
+description: Ballerina has powerful support for Electronic Data Interchange (EDI) formats. In healthcare, this means out-of-the-box handling of X12, 834s, 837s, and more regional-specific standards via native schema mapping. Go from EDI formats to Ballerina Records and back; Do Data Mapping on that, and you’re off to the races!
```

Comment:
region-specific

Reply:
Why upper case in "Do Data Mapping"?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/9506/commits/114a2dd9512f0241a3961fc7d28b263351f0bc43